<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 24px;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Registrace dokončena</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_result">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>300</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Uložit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Zde můžete uložit výsledné CSV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="main_menu">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Zpět do menu</string>
    </property>
   </widget>
   <widget class="QLabel" name="file_saved_label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: green;</string>
    </property>
    <property name="text">
     <string>Soubor uložen</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
